From d16c6cfdc6262dc3b89586ed681b67b2a936f0b6 Mon Sep 17 00:00:00 2001
From: Kylie McClain <kylie@somas.is>
Date: Sat, 30 Nov 2019 19:24:34 -0500
Subject: [PATCH 219/221] linux-clang/qplatformdefs: fix building with musl
 libc

This is basically a duplicate of the commit which fixed building with
musl, but on linux-g++, 813f468a14fb84af43c1f8fc0a1430277358eba2.

Change-Id: I399005ac6947ba3f2b4ed5087472cd9d54a0850d
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
(cherry picked from commit a421e40984a61b7051d045832af07988d692f67d)
---
 mkspecs/linux-clang/qplatformdefs.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/mkspecs/linux-clang/qplatformdefs.h b/mkspecs/linux-clang/qplatformdefs.h
index a818d973f0..34cd665883 100644
--- a/mkspecs/linux-clang/qplatformdefs.h
+++ b/mkspecs/linux-clang/qplatformdefs.h
@@ -81,11 +81,7 @@
 
 #undef QT_SOCKLEN_T
 
-#if defined(__GLIBC__) && (__GLIBC__ >= 2)
 #define QT_SOCKLEN_T            socklen_t
-#else
-#define QT_SOCKLEN_T            int
-#endif
 
 #if defined(_XOPEN_SOURCE) && (_XOPEN_SOURCE >= 500)
 #define QT_SNPRINTF             ::snprintf
-- 
2.33.0

